data/sql/updates/pending_db_world/rev_1634939032065262806.sql

```diff
@@ -0,0 +1,102 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1634939032065262806');
+
+-- *** Quest "Last rites" ***
+
+-- Remove Quest SAI
+UPDATE `creature_template` SET `AIName`='' WHERE `entry` IN (26170,26203,25301,25250,25251);
+DELETE FROM `smart_scripts` WHERE `entryorguid` IN (26170,26203,25301,25250,25251);
```

Comment:
@malcrom we should add `source_type` here to avoid deleting unwanted rows

Reply:
Will update

Reply:
Fixed